fast_llm/layers/language_model/head.py

```diff
@@ -369,30 +377,51 @@ def _logits_cross_entropy_forward_backward(
             dpo_loss, dpo_grad = None, None
 
         if lm_target is not None:
+            distil_ntp_loss_factor = 1.0
+            if self._config.distillation_model is not None:
```

Comment:
`distil_ntp_loss_factor = 1.0 if self._config.distillation_model is None else distil_ntp_loss_factor `

Reply:
Not sure that's the best option, lm loss has nothing to do with distillation so I recommend:
* Use variable `language_model_loss_factor : float = None`
* Set implicit default in validation to 1.0 or 0.0
* Always use this factor here.